fbchat/_client.py

```diff
@@ -2727,6 +2727,17 @@ def getThreadIdAndThreadType(msg_metadata):
                 msg=delta,
             )
 
+        # New pending message
+        elif delta_class == "ThreadFolder" and delta.get("folder") == "FOLDER_PENDING":
+            threadId = delta["threadKey"]["otherUserFbId"]
+            self.onPendingMessage(
+                author_id=threadId,
+                thread_id=threadId,
+                ts=now(),  # Also h4ck3r
+                metadata=metadata,
+                msg=delta,
+            )
```

Comment:
Only add relevant fields! The reason the fields still exist for the other methods is to be backwards compatible.